src/support/ios.c

```diff
@@ -831,6 +831,66 @@ size_t ios_copyuntil(ios_t *to, ios_t *from, char delim)
     return total;
 }
 
+//Copy until '\r', '\n' or '\r\n'
+size_t ios_copyline(ios_t *to, ios_t *from, int chomp)
```

Comment:
OK, it's just that I found a `bool_t` in the same file, but I have no idea what's the most common convention.